src/Action/NotifyAction.php

```diff
@@ -25,15 +31,17 @@ public function __construct(
         private readonly RequestStack $requestStack,
         private readonly SignatureResolverInterface $signatureResolver,
     ) {
-        $this->request = $requestStack->getCurrentRequest();
+        $this->request = $requestStack->getCurrentRequest() ?? new Request();
```

Comment:
I'm not sure but probably if request doesn't exists then exceptiion should be throws.

Reply:
Exactly, if there is no request, we cannot create a new one here, this is the action responsible for setting the transaction status

Reply:
I implemented a fix in the commit which now throws an exception.